charactersheet/charactersheet/index.html

```diff
@@ -216,14 +216,15 @@ <h4 class="modal-title" id="exportModalLabel" data-dismiss="modal">Export</h4>
           data-dismiss="modal">Create a new character</h4>
       </div>
       <div class="modal-body">
-        <p>If you play more than one character per game, or if you
-        play multiple games, you're in luck! You can have
-        multiple characters and switch between them at any time!</p>
+        <p>If you play more than one character per game,
+					run multiple campaigns, or do a little of both, you're
+					in luck! Create a new character or campaign and then use
+					the "Characters and Games" menu to switch between them.</p>
       </div>
       <div class="modal-footer">
         <button data-bind="click: showWizard"
           data-dismiss="modal" type="submit"
-          class="btn btn-primary">Make a new character
+          class="btn btn-primary">Create a new character
```

Comment:
Since that might get kinda big, should we just call it? __Get Started!__ or something?